actors/builtin/miner/miner_test.go

```diff
@@ -1764,16 +1829,18 @@ func TestChangeWorkerAddress(t *testing.T) {
 		return rt, actor
 	}
 
-	t.Run("successfully change a worker address", func(t *testing.T) {
+	t.Run("successfully change ONLY the worker address", func(t *testing.T) {
```

Comment:
After the change requested above, replace this with a test that changing to an empty array removes all control addresses.

Reply:
This is done.